R/tt_pos_and_access.R

```diff
@@ -327,52 +327,90 @@ setMethod("tt_at_path<-", c(tt = "VTableTree", value = "TableRow"),
 #'   \code{FALSE}
 #' @param \dots Includes
 #' \describe{
-#' \item{\emph{keep_topleft}}{ logical(1) (\code{[} only) Should the 'top-left'
-#' material for the table be retained after subsetting. Defaults to \code{NA},
-#' which retains the material if all rows are included (ie subsetting was by
-#' column), and drops it otherwise.}
-#' \item{\emph{keep_titles}}{logical(1) Should title and non-referential footer
-#' information be retained. Defaults to \code{FALSE}}
+#' \item{\emph{keep_topleft}}{logical(1) (\code{[} only) Should the `top-left`
+#' material for the table be retained after subsetting. Defaults to `TRUE` if 
+#' all rows are included (i.e. subsetting was by column), and drops it otherwise.}
+#' \item{\emph{keep_titles}}{logical(1) Should title information be retained. Defaults to \code{FALSE}.}
+#' \item{\emph{keep_fnotes}}{logical(1) Should non-referential footer
+#' information be retained. Defaults to \code{keep_titles}.}
 #' \item{\emph{reindex_refs}}{logical(1). Should referential footnotes be
 #' re-indexed as if the resulting subset is the entire table. Defaults to
-#' \code{TRUE}}
+#' \code{TRUE}.}
 #' }
 #' @param value Replacement value (list, `TableRow`, or `TableTree`)
+#' 
 #' @return a \code{TableTree} (or \code{ElementaryTable}) object, unless a
 #'   single cell was selected with \code{drop=TRUE}, in which case the (possibly
 #'   multi-valued) fully stripped raw value of the selected cell.
-#' @exportMethod [<-
+#' 
+#' @details the standard subsetting drops the information about title, subtitle,
```

Comment:
Please use "by default" instead of "the standard" (here and generally) as its more concise and accurate in english